src/ellip.jl

```diff
@@ -0,0 +1,196 @@
+using Base.Math: @horner
+
+#Using piecewise approximation polynomial as given in
+#'Fast Computation of Complete Elliptic Integrals and Jacobian Elliptic Functions'
+#                          Fukushima, Toshio. (2014). F09-FastEI. Celest Mech Dyn Astr
+#                                                            DOI 10.1007/s10569-009-9228-z
+#Link : https://pdfs.semanticscholar.org/8112/c1f56e833476b61fc54d41e194c962fbe647.pdf
+#
+#For m<0 , followed  Fukushima, Toshio. (2014). Precise, compact, and fast computation of complete elliptic integrals by piecewise minimax rational function approximation. Journal of Computational and Applied Mathematics. 282. 10.13140/2.1.1946.6245.   
+# Link:  https://www.researchgate.net/profile/Toshio_Fukushima/publication/267330394_Precise_compact_and_fast_computation_of_complete_elliptic_integrals_by_piecewise_minimax_rational_function_approximation/links/544b81a40cf2d6347f43074f/Precise-compact-and-fast-computation-of-complete-elliptic-integrals-by-piecewise-minimax-rational-function-approximation.pdf?origin=publication_detail
+#Also suggested in this paper that we should consider domain only from (-inf,1].
+#Wiki : https://en.wikipedia.org/wiki/Elliptic_integral 
+#DLMF:  https://dlmf.nist.gov/19
```

Comment:
Which function in particular from the DLMF? Can you reference the particular equation where it is defined Also, this would be better in the docs than a comment.

Reply:
Okay .. I'll update the index.md then with DLMF link.